### DISCUSSION TITLE
Warning when using Tagify preset with Nuxt

### DISCUSSION THREAD

**Discussion by dikesh** [unocss/unocss#2551] [Q&A]
When enabling Tagify preset with Nuxt 3, the style is applied correctly but I am receiving the warning as below

**Tag and style applied**

![image](https://user-images.githubusercontent.com/1618526/235077433-dbb3551a-0883-483d-8078-92cbc613a07d.png)

**Warning**

![image](https://user-images.githubusercontent.com/1618526/235077620-b167c52f-791f-4460-8adb-dab38c510bd4.png)

How can I resolve this?

Here is the [Stackblitz demo](https://stackblitz.com/edit/nuxt-starter-gwrmsr?file=pages/index.vue)

**Comment by genu** [comment]
Is there a workaround for this? I'm not exactly sure where this warning is thrown (nuxt, or vue compiler). We also don't want to just disable that warning altogether.
